guides/v2.3/graphql/reference/store-config.md

```diff
@@ -5,83 +5,43 @@ title: Store endpoint
 
 The `Store` endpoint defines information about a store's configuration. You can query a non-default store by [changing the header]({{ page.baseurl }}/graphql/send-request.html) in your GraphQL request.
 
-## Query structure
+## Syntax
 
 `storeConfig: StoreConfig`
 
-### Supported storeConfig attributes
+### Store config attributes
 
 Use the `storeConfig` attributes to retrieve information about the store's configuration; such as, locale, currency codes, and secure and unsecure URLs.
 
-Attribute |  Data Type | Description
---- | --- | ---
-`base_currency_code` | String | The code representing the currency in which Magento processes all payment transactions, such as `USD`
-`base_link_url` | String | A fully-qualified URL that is used to create relative links to the `base_url`
-`base_static_url` | String | The fully-qualified URL that specifies the location of static view files, such as `http://magentohost.example.com/pub/static/`
-`base_media_url` | String | The fully-qualified URL that specifies the location of user media files, such as `http://magentohost.example.com/pub/media/`
-`base_url` | String | The store's fully-qualified base URL, such as `http://magentohost.example.com/`
-`code` | String | A unique identifier for the store
-`default_display_currency_code` | String | The code representing the currency displayed on the store, such as `USD`
-`id` | Integer | The ID number assigned to the store
-`locale` | String | The store's locale, such as `en_US`
-`secure_base_link_url` | String | A secure fully-qualified URL that is used to create relative links to the `base_url`
-`secure_base_media_url` | String | The secure fully-qualified URL that specifies the location of user media files, such as `https://magentohost.example.com/pub/media/`
-`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
-`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
-`timezone` | String | The store's time zone, such as `America/Chicago`
-`website_id` | Integer | The ID number assigned to the parent website  
-`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
-{:style="table-layout:auto;"}
-
-### Supported theme attributes
```

Comment:
I want to keep the general layout of the topic the same for now.  I want to keep all the reference info in one section and examples in another. We might change the layout to agree with what you've done here, but it would be best to change all the documentation at the same time.

Reply:
**fixed**

Could you please review?

Thank you, @keharper !